triage/2024-05-21.md

```diff
@@ -0,0 +1,134 @@
+# 2024-05-21 Triage Log
+
+Fairly quiet week with the except of a very large improvement coming from the switch to `rust-lld` on nightly Linux. This can have very large impacts on benchmarks where linking dominates the build time (e.g., `hello-world`). Aside from that change, there were a few small regressions that were either deemed worth it or are still being investigated.
```

Comment:
(also there's a typo)

how about something like this?

```suggestion
Fairly quiet week with the exception of a very large improvement coming from the switch to `rust-lld` on nightly Linux. This can have very large impacts on benchmarks where linking dominates the build time (e.g., `ripgrep`, `exa`, small binaries like `hello-world`). Aside from that change, there were a few small regressions that were either deemed worth it or are still being investigated.
```